lib/state/actions/pokemon_list_actions.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex/state/app_state.dart';
+
+class GetPokemonList extends ReduxAction<AppState> {
```

Comment:
also add documentation above your Action classes and state its purpose why that action is created 🙂